.ci/azure-pipelines.yml

```diff
@@ -23,6 +23,11 @@ jobs:
     pool:
       vmImage: 'ubuntu-latest'
 
+    variables:
+      - name: JELLYFIN_VERSION
+        value: replace(variables['Build.SourceBranch'], 'refs/tags/', '')
+        condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/v')
+
```

Comment:
```suggestion
```

Reply:
as mentioned by @EraYaN and me in the conversation above, this should only be set in the job that needs it and that anyway is already only run on condition

Reply:
See my other comment on why we need it for the app but not the apiclient